src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -342,8 +341,6 @@ public void It_implicitly_defines_compilation_constants_for_the_target_framework
                     if (targetFramework.Contains(",Version="))
                     {
                         //  We use the full TFM for frameworks we don't have built-in support for targeting, so we don't want to run the Compile target
```

Comment:
I think you want to delete this comment, as you deleted the code it referred to.